controllers/userQuizQuestion.js

```diff
@@ -125,11 +126,47 @@ const getUserQuizQuestionOptionsByQuestion = async (question) => {
     return packOptions(answers)
 }
 
+const getUserAnswerIDs = async (userQuiz) => {
+    const userAnswers = await getUserQuizQuestions(userQuiz)
+
+    const userAnswerIDsRequest = userAnswers.map(async (userAnswer) => {
+        let userAnswerObj = userAnswer.userAnswerObj
+        return await userAnswerObj.answer.get()
+    });
+
+    let userAnswerIDs = []
+    
+    Promise.all(userAnswerIDsRequest).then((answerIDObjs) => {
+        answerIDObjs.forEach((obj) => {
+            userAnswerIDs.push(obj.id)
+        })
+    })
+
+    return userAnswerIDs
+}
```

Comment:
me don't like. 

also, I hate people calling stuff obj. Like either make it `answerIDObj` or just use a single letter `a` (not perfect but me take). like obj can mean anything.

prefer:
```js
// example don't use directly

const userAnswerIDs = await Promise.all(userAnswer.map(async (userAnswer) => {
	let userAnswerObj = userAnswer.userAnswerObj
   	const answer = await userAnswerObj.answer.get() 
   	
	return answer.id
}))

```